nilearn/connectivity/connectivity_matrices.py

```diff
@@ -0,0 +1,341 @@
+import warnings
+from math import sqrt
+
+import numpy as np
+from scipy import linalg
+
+from sklearn.base import BaseEstimator, TransformerMixin, clone
+from sklearn.covariance import EmpiricalCovariance
+from .._utils.extmath import is_spd
+
+
+def _check_matrix(matrix, prop):
+    """Raise a ValueError if the input matrix does not satisfy the property.
+
+    Parameters
+    ----------
+    matrix : numpy.ndarray
+        Input array.
+
+    prop : {'square', 'symmetric', 'spd'}
+        Property to check.
+    """
+    if prop == 'square':
+        if matrix.ndim != 2 or (matrix.shape[0] != matrix.shape[-1]):
+            raise ValueError('Expected a square matrix, got array of shape' +
```

Comment:
You don't need the + here. Strings are concatenated as long as you have parentheses around them, e.g.:

```
 In [1]: ('some content '
   ...:  'and some more content')
Out[1]: 'some content and some more content'
```